include/envoy/server/hot_restart.h

```diff
@@ -20,11 +22,6 @@ class Instance;
  */
 class HotRestart {
```

Comment:
Can we remove `statsAllocator()` from this interface? I don't think we need it anymore since we are only use a heap allocator which hopefully can be created somewhere more obvious than from this interface?

Reply:
Done.